test/unit/PanoImageRenderer/ImageLoader.spec.js

```diff
@@ -170,19 +206,18 @@ describe("ImageLoader", function() {
 			// When
 			let inst;
 			const imgObj = new Image();
-
-			imgObj.onload = function() {
+			imgObj.addEventListener("load", function() {
```

Comment:
@happyhj `imgObj.onload` was intended code to test "it should handle `onload` handler.".